Description: adjust javac compile version
 The compile version provided by upstream are not suitable for the currently
 distributed Java packages anymore.
Author: Lars Kruse <devel@sumpfralle.de>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile.config
+++ b/Makefile.config
@@ -159,7 +159,7 @@ CHGRP      := chgrp
 
 # Java compiler stuff - only needed on the buildhost
 JC         := javac
-JFLAGS     := -g -source 1.5 -target 1.5 -Xlint
+JFLAGS     := -g -source 1.7 -target 1.7 -Xlint
 JAR        := jar
 
 # Check if the java compiler works
